### ISSUE TITLE
Layout position of SlideShowView

### ISSUE THREAD

**Issue by dantp-ai** [spiritinlife/EasySlideShow4Android#3]
I have a RelativeLayout as parent and want to position SlideShowView in the center of the parent. However this does not seem to work?

**Comment by spiritinlife** [comment]
```
<gr.spiritinlife.EasySlideShow.SlideShowView
        android:id="@+id/slideShow"
        android:layout_width="match_parent"
        android:layout_height="170dp"
        android:clickable="true"
       android:layout_centerInParent="true"/>
```

You have something like the above and it does not get centered ?
If not please provide some code.

**Comment by dantp-ai** [comment]
Yes, I define `SlideShowView` layout like that. I put it inside a `RelativeLayout`:

```
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:background="@android:color/black">
</RelativeLayout>
```

In Android Studio Design section, it seems to look correct(Fig. 1), however on my phone it looks different(Fig. 2)

**(Fig. 1)**
![layout-2015-10-11-092055](https://cloud.githubusercontent.com/assets/1534513/10415532/17519452-6ffa-11e5-819e-6ccd3716fff2.png)

**(Fig.2)**
![screenshot_2015-10-11-09-26-45](https://cloud.githubusercontent.com/assets/1534513/10417035/680ac77c-702e-11e5-88b8-f216861f3242.png)

**Comment by spiritinlife** [comment]
I cannot reproduce it.
I use this code and it seems fine 

```
<?xml version="1.0" encoding="utf-8"?>
<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
              android:orientation="vertical"
              android:layout_width="match_parent"
              android:layout_height="match_parent">


    <gr.spiritinlife.EasySlideShow.SlideShowView
            android:id="@+id/slideShow"
            android:layout_width="match_parent"
            android:layout_height="170dp"
            android:clickable="true"
            android:layout_centerInParent="true"/>


</RelativeLayout>
```

Sorry i cannot be of more help/

**Comment by dantp-ai** [comment]
Does it have something to do with your implementation of `newImageViewInstance()` ?

**Comment by spiritinlife** [comment]
I don't see something wrong with it . But the whole thing is not correct. I suggest you not use this for any of your apps. At the moment i am working on a new one that will be more stable. Maybe by the end of the day i will have something.

**Comment by dantp-ai** [comment]
That would be awesome. \m/

**Comment by spiritinlife** [comment]
@plopd Check the new release that i pushed. It is still a pre release but i believe it is in the right direction.
- Check the updated readme which is built upon your issue.

Tell if everything works this time
Cheers

**Comment by spiritinlife** [comment]
Closing it

**Comment by dantp-ai** [comment]
I managed to get the new version running. However, if I use your example, and I would expect that the 'SlideShowView' layout is centered on the screen, it is still positioned on top.

**Comment by spiritinlife** [comment]
Here is a screenshot. Could you send me the specs of your testing device to try to emulate it ?
The following screenshot is from a nexus 4 running android 5+
![screenshot_2015-10-21-18-15-16](https://cloud.githubusercontent.com/assets/6434983/10641128/90175d44-7820-11e5-9f58-ade12c53b32b.png)

**Comment by dantp-ai** [comment]
Android 4.2.2 Samsung Galaxy S2

**Comment by spiritinlife** [comment]
I have tested in genmotion emulator
With Samsung Galaxy S2 4.1.1 and Samsung Galaxy S3 4.2.2 and it looks correct.

Can you share some code?
At the moment , i do not see how this is not working for you.
Can you run your project on an emulator also to confirm that is not device specific ?

**Comment by dantp-ai** [comment]
Yes, I will look into it.
